qiskit/chemistry/components/variational_forms/uccsd.py

```diff
@@ -124,9 +146,20 @@ def __init__(self, num_qubits, depth, num_orbitals, num_particles,
                                           sq_paulis, sq_list, tapering_values, and cliffords
             shallow_circuit_concat (bool): indicate whether to use shallow (cheap) mode for
                                            circuit concatenation
+            method_singles (str): specify the single excitation considered. 'alpha', 'beta',
+                                  'both' only alpha or beta spin-orbital single excitations or
+                                  both (all of them)
+            method_doubles (str): specify the single excitation considered. 'ucc' (conventional
+                                  ucc), succ (singlet ucc), succ_full (singlet ucc full)
+            excitation_type (str): specify the excitation type 'sd', 's', 'd' respectively
+                                   for single and double, only single, only double excitations.
+            same_spin_doubles (bool):, enable double excitations of the same spin.
+            force_no_tap_excitation (bool): keep all the excitation regardless if tapering is used.
```

Comment:
Maybe rather than 'force_no_tap_excitation' how about 'ignore_commute' which when True will include all tapered excitation operators even when they do not commute with symmetry. Default, i.e. False, is to not include these. (The description is what happens I am just trying to make name a bit better - and change the description to be a bit better)

Reply:
Yes, I agree on this one. Perhaps 'ignore_excitation_tapering' ? A bit long but mentions what it acts on.

Reply:
My understanding is that we always use tapered excitation op if there are Z2Symmetries. The difference seems to be the old code drops them if it did not commute with the symmetries (symm_commuting test). This new flag causes them to be always be included and hence ignores the symm commuting

Reply:
Yes indeed, that is what I intended. Just so all operators corresponding to excitations are included in any case.